app/controllers/observer_controller/create_and_edit_observation.rb

```diff
@@ -42,6 +50,10 @@ def create_observation
     end
   end
 
+  ##############################################################################
+
+  private
```

Comment:
`private` sections added in order to clarify which methods are (not) part of the public interface